webcompat/static/css/development/components/navbar.css

```diff
@@ -68,6 +68,11 @@
     vertical-align: middle;
   }
 
+  /* .is-hidden class added when add-on is installed */
+  .wc-Navbar .is-hidden {
+    display: none;
+  }
+
```

Comment:
something like <a class="wc-Navbar-link is-hidden js-Navbar-link" ?

Yeah, it will only get added to that one element. I'm using `.js-Navbar-link` to find the element and add `is-hidden` class:  see https://github.com/webcompat/webcompat-reporter-extensions/blob/3abdfab2af19f55d1e372272d8f743b88d2ae61f/firefox-webext/content.js